Model/SyncStatus.php

```diff
@@ -213,12 +213,10 @@ private function markPaymentExpired($order)
      */
     private function markPaymentReversed($order)
     {
-        $order->addStatusHistoryComment(__('Omise: Payment reversed. (manual sync).'));
-
-        if ($order->getState() != Order::STATE_CANCELED) {
-            $order->setState(Order::STATE_CANCELED)->setStatus(Order::STATE_CANCELED);
-        }
-
+        $this->cancelOrderInvoice($order);
+        $order->registerCancellation(
+            __('Omise: Payment reversed. (manual sync).')
+        )->save();
         $order->save();
```

Comment:
just wondering is 
```php 
        $order->registerCancellation(
            __('Omise: Payment reversed. (manual sync).')
        )->save();
```
and  
```php 
$order->save();
```  
is not the same save method right 🤔 ?

Reply:
Nice catch. Thanks for pointing this out. They are the same method. I removed the duplicate `save`. Thanks.